API.Microsoft.Extensions.Compliance.Redaction/NoDocs/Microsoft.Extensions.Compliance.Redaction/HmacRedactorOptions.cs

```diff
@@ -0,0 +1,19 @@
+// Assembly 'Microsoft.Extensions.Compliance.Redaction'
+
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using Microsoft.Shared.Data.Validation;
+
+namespace Microsoft.Extensions.Compliance.Redaction;
+
+[Experimental("EXTEXP0002", UrlFormat = "https://aka.ms/dotnet-extensions-warnings/{0}")]
+public class HmacRedactorOptions
+{
+    public int? KeyId { get; set; }
```

Comment:
I think that for a large system it might be hard to have globally unique IDs that are just ints. As this is just to identify key used in encryption process, maybe it could be a base64 string as well?

Reply:
Switch to string.